bokeh/models/map_plots.py

```diff
@@ -84,3 +84,8 @@ def _check_no_data_renderers(self):
     """)
 
     border_fill_color = Override(default="#ffffff")
+
+    api_key = String(help="""
+    Google Maps API requires an API key. See https://developers.google.com/maps/documentation/javascript/get-api-key
+    for more information on how to obtain your own.
+    """)
```

Comment:
we should add a new validation error for this python side so that the problem shows up as early as possible